patches/chromium/chore_add_electron_objects_to_wrappablepointertag.patch

```diff
@@ -8,20 +8,20 @@ electron objects that extend gin::Wrappable and gets
 allocated on the cpp heap
 
 diff --git a/gin/public/wrappable_pointer_tags.h b/gin/public/wrappable_pointer_tags.h
-index 80ec409efe1635390887d1324be661643818abff..2e20b63d1fca56efb43c18d0fa04b1e2b4cf339a 100644
+index a507d1d837ab3ec2b2d3ae7978d9d410ab2ec2d1..b9e9f4101fcf9065e1de22c0e8f5680efe831708 100644
 --- a/gin/public/wrappable_pointer_tags.h
 +++ b/gin/public/wrappable_pointer_tags.h
-@@ -66,7 +66,12 @@ enum WrappablePointerTag : uint16_t {
-   kTextInputControllerBindings,         // content::TextInputControllerBindings
-   kWebAXObjectProxy,                    // content::WebAXObjectProxy
-   kWrappedExceptionHandler,             // extensions::WrappedExceptionHandler
+@@ -72,7 +72,12 @@ enum WrappablePointerTag : uint16_t {
+   kTextInputControllerBindings,  // content::TextInputControllerBindings
+   kWebAXObjectProxy,             // content::WebAXObjectProxy
+   kWrappedExceptionHandler,      // extensions::WrappedExceptionHandler
 -  kLastPointerTag = kWrappedExceptionHandler,
-+  kElectronApp,                         // electron::api::App
-+  kElectronDebugger,                    // electron::api::Debugger
-+  kElectronEvent,                       // gin_helper::internal::Event
-+  kElectronNetLog,                      // electron::api::NetLog
-+  kElectronSession,                     // electron::api::Session
-+  kLastPointerTag = kElectronEvent,
++  kElectronApp,                  // electron::api::App
++  kElectronDebugger,             // electron::api::Debugger
++  kElectronEvent,                // gin_helper::internal::Event
++  kElectronNetLog,               // electron::api::NetLog
++  kElectronSession,              // electron::api::Session
++  kLastPointerTag = kElectronSession,
```

Comment:
I think this was a previous error. kLastPointerTag pointed at kElectronEvent instead of the actual last value kElectronSession